stringstream.js

```diff
@@ -27,6 +27,9 @@ StringStream.prototype.write = function(data) {
     this.emit('error', err)
     return false
   }
+  if (typeof data !== 'string') {
```

Comment:
This isn't right. `data` can be a `Buffer`.

Did you mean to check `this.fromEncoding` instead?

Reply:
Nope, I'm wrong – in any case, a better check is to ensure that data isn't a number

Reply:
ummm.... !isNan(data)